master/getting-started/bare-metal/bare-metal.md

```diff
@@ -427,3 +430,280 @@ an ingress rule allowing access *to* port 999 and an egress rule allowing
+  > in the forwards direction, and if so it will have the effect of forwarding
+  > the packet to the workload without any connection tracking.  But then, in
+  > the reverse direction, there will be no conntrack state for the return
+  > packets to match, and there is no application of any egress rules that may
+  > be defined by the untracked policy - so unless the workload's policy
+  > specifically allows the relevant source IP, the return packet will be
+  > dropped.  That is the same overall result as if there was no untracked
+  > policy at all, so in practice it is as if untracked policies do not apply
+  > to this flow.
+
+- Pre-DNAT policies apply.
+
+- Normal policies do not apply - by design, because Calico enforces the
+  destination workload's ingress policy in this case.
+
+For packets that arrive on a host interface and are destined for a local
+server in the host namespace:
+
+- Untracked, pre-DNAT and normal policies all apply.
+
+- If a packet is explicitly allowed by untracked policy, it skips over any
+  pre-DNAT and normal policy.
+
+- If a packet is explicitly allowed by pre-DNAT policy, it skips over any
+  normal policy.
+
+For packets that arrive on a host interface (A) and are forwarded out of the
+same or another host interface (B):
+
+- Untracked policies apply, for both host interfaces A and B, but only the
+  ingress rules that are defined in those policies.  The forwards direction is
+  governed by the ingress rules of untracked policies that apply to interface
+  A, and the reverse direction is governed by the ingress rules of untracked
+  policies that apply to interface B, so those rules should be defined
+  symmetrically.
+
+- Pre-DNAT policies apply, specifically the ingress rules of the pre-DNAT
+  policies that apply to interface A.  (The reverse direction is allowed by
+  conntrack state.)
+
+- Normal policies apply, specifically the ingress rules of the normal policies
+  that apply to interface A, and the egress rules of the normal policies that
+  apply to interface B.  (The reverse direction is allowed by conntrack state.)
+
+- If a packet is explicitly allowed by untracked policy, it skips over any
+  pre-DNAT and normal policy.
+
+- If a packet is explicitly allowed by pre-DNAT policy, it skips over any
+  normal policy.
+
+For packets that are sent from a local server (in the host namespace) out of a
+host interface:
+
+- Untracked policies apply, specifically the egress rules of the untracked
+  policies that apply to the host interface.
+
+- Pre-DNAT policies do not apply.
+
+- Normal policies apply, specifically the egress rules of the normal policies
+  that apply to that host interface.
+
+For packets that are sent from a local workload out of a host interface:
+
+- No host endpoint policies apply.
+
+## Pre-DNAT policy: a worked example
+
+Imagine a Kubernetes cluster, that its administrator wants to secure as much as
```

Comment:
This is a great example but a diagram would be a great help here, it was at the limit of my mental "stack" to follow!

It might help to point out that you're talking about a VIP that routes to several hosts for the load balancer; I didn't grok that at first; left me wondering why several hosts could have the same IP in the dest.